src/util/syncbuffer.cpp

```diff
@@ -37,7 +37,13 @@ SyncBuffer::~SyncBuffer() {} // NOLINT
 void SyncBuffer::encode(const std::string& s) {
     ghoul_assert(_encodeOffset + sizeof(char) * s.size() + sizeof(int32_t) < _n, "");
```

Comment:
This assert should be removed now right? It would trigger and exit the application even though larger buffer size is actually supported below.

Reply:
Correct, this line has been removed.